tests/testdata/sdg/tonsil_task.yaml

```diff
@@ -0,0 +1,6 @@
+dataset_kwargs:
+  data_files:
+    test: tonsil_data.jsonl
```

Comment:
This is usually the absolute path to the `.jsonl` file for the task, but in our case a relative path will work as long as we can assume the dir the test is going to run out of. Take a look at https://github.com/instructlab/instructlab/blob/main/tests/testdata/mmlu_branch/tonsils_task.yaml#L3

In our case I think it's fine to assume running test_mmlubranch.py will be done out of the root of the eval repo, so the path here needs to be `test: tests/testdata/sdg/tonsil_data.jsonl`